Common/MenuMeterPowerMate.m

```diff
@@ -154,24 +154,27 @@ - (id)init {
 										   matchingDict,
 										   DeviceMatched, (__bridge void *)(self), &deviceMatchedIterator);
 	if (err != KERN_SUCCESS) {
+        CFRelease(terminatedDict);
```

Comment:
tab/space mixup here it seems

Reply:
not really, just aligned method arguments vertically.